Tests/PackageLoadingTests/PD_6_2_LoadingTests.swift

```diff
@@ -0,0 +1,100 @@
+//===----------------------------------------------------------------------===//
+//
+// This source file is part of the Swift open source project
+//
+// Copyright (c) 2025 Apple Inc. and the Swift project authors
+// Licensed under Apache License v2.0 with Runtime Library Exception
+//
+// See http://swift.org/LICENSE.txt for license information
+// See http://swift.org/CONTRIBUTORS.txt for the list of Swift project authors
+//
+//===----------------------------------------------------------------------===//
+
+import Basics
+import PackageLoading
+import PackageModel
+import _InternalTestSupport
+import Testing
+
+struct PackageDescription6_2LoadingTests {
```

Comment:
praise: It's great to a a new test suite being written in Swift Testing.  thanks for this.